app/services/cached_commodity_service.rb

```diff
@@ -1,5 +1,5 @@
 class CachedCommodityService
-  CACHE_VERSION = TradeTariffBackend.permutations? ? 2 : 1
+  CACHE_VERSION = 2
```

Comment:
maybe we could even drop this cache_version? 🤔

Reply:
No - the whole point of the CACHE_VERSION is it gives us an easy way to evolve the invalidate the cache contents in the future without needing to co-ordinate manual cache invalidations together with deploys - this way a developer can just bump the CACHE_VERSION number in their PR and whenever a feature which changes the API contents gets deployed and the invalidation just happens automatically